merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertSubsystemsAction.java

```diff
@@ -0,0 +1,67 @@
+package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
+
+import org.intellij.lang.annotations.Language;
+
+import java.sql.BatchUpdateException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/*package-local*/ final class InsertSubsystemsAction implements AutoCloseable {
+  private static final @Language("SQL") String insertSQL = """
+    insert into tags.tags (name, owner, color) values (?, 'Mission Model', '#000000')
+    on conflict (name) do nothing;
+""";
+
+  private static final @Language("SQL") String findTagsSQL = """
+    select name, id
+    from tags.tags as t;
+    """;
+  // where t.name in ?; TODO Check if we need this where clause since it would make it more efficient to only grab tags
+  //                       associated with a subsystem instead of all tags
```

Comment:
This statement isn't used in the action (and if anything should be in a separate action)